FunctionalTableData/HostCell.swift

```diff
@@ -95,7 +95,10 @@ public struct HostCell<View, State, Layout>: CellConfigType where View: UIView,
 	
 	public func isEqual(_ other: CellConfigType) -> Bool {
 		if let other = other as? HostCell<View, State, Layout> {
-			return state == other.state
+			var isEqual = key == other.key
+			isEqual = isEqual && state == other.state
+			isEqual = isEqual && style == other.style
+			return isEqual
```

Comment:
I think it's ok to just `&&` the conditions together now.

Reply:
Can you describe the issue that led you to here. The keys and styles are compared separately by TableSectionChangeSet so I would have thought that comparing state was enough here.

Reply:
Main reason is that if we want to equate `CellConfigType`s outside of the `TableSectionChangeSet`, it wouldn't take `style` into account. Though I'm currently revisiting our use case to see if we really do need to equate these.